README.md

```diff
@@ -50,6 +50,7 @@ currently supports the following ansible roles:
 | [setup_patroni](roles/setup_patroni/README.md)                                           | Set up for patroni managed cluster.                                                                                                                                                                    |
 | [setup_pgpool2](roles/setup_pgpool2/README.md)                                           | Set up Pgpool-II connection pooler/load balancer.                                                                                                                                                      |
 | [setup_replication](roles/setup_replication/README.md)                                   | Set up the data replication (synchronous/asynchronous).                                                                                                                                                |
+| [setup_repmgr](roles/setup_repmgr/README.md)                                             | Set up Repmgr for PostgreSQL/EPAS HA cluster.                                                                                                                                                          |
```

Comment:
Should this be `Set up Repmgr for PostgreSQL HA cluster` instead of including the EPAS HA cluster? We can always change in another commit but it appears that repmgr is only setup to work with PG.

Reply:
You are right. Fixed.